demo/src/main/java/com/orange/ods/demo/ui/components/dialogs/ComponentDialog.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.dialogs
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.rememberBottomSheetScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import com.orange.ods.compose.component.button.OdsButton
+import com.orange.ods.compose.component.button.OdsButtonContainedStyle
+import com.orange.ods.compose.component.dialog.OdsAlertDialog
+import com.orange.ods.compose.text.OdsTextBody1
+import com.orange.ods.demo.R
+import com.orange.ods.demo.ui.components.buttons.fullWidthButton
+import com.orange.ods.demo.ui.components.utilities.ComponentCustomizationBottomSheetScaffold
+import com.orange.ods.demo.ui.components.utilities.clickOnElement
+import com.orange.ods.demo.ui.utilities.composable.SwitchListItem
+
+@ExperimentalMaterialApi
+@Composable
+fun ComponentDialog() {
```

Comment:
The demo screen displays only one dialog so I suppose that's why the name is `ComponentDialog`.

However other method names are all using plural. Should we rename this `ComponentDialogs` ?